src/api/Field.ts

```diff
@@ -0,0 +1,148 @@
+export default class Field {
+  size: number;
+
+  color: string;
+
+  ctx: CanvasRenderingContext2D;
+
+  tiles: Record<string, number>[];
+
+  numbers: number[];
+
+  // eslint-disable-next-line max-len
+  constructor(fieldSize: number, fieldColor: string, ctx: CanvasRenderingContext2D, tiles: Record<string, number>[], numbers: number[]) {
```

Comment:
```suggestion
  constructor(
    fieldSize: number,
    fieldColor: string,
    ctx: CanvasRenderingContext2D,
    tiles: Record<string, number>[],
    numbers: number[]
  ) {
```
В целом лучше в случае, когда параметров много, использовать options object, в таком случае и читаемость повышается и в самом коде при создании нового инстанса глазу приятней должно быть, как мне кажется:
```suggestion
  type FieldProps = {
    fieldSize: number;
    fieldColor: string;
    ctx: CanvasRenderingContext2D;
    tiles: Record<string, number>[];
    numbers: number[];
  };
  
  constructor({ fieldSize, fieldColor, ctx, tiles, numbers }: FieldProps) {
```